lib/streamlit/delta_generator.py

```diff
@@ -145,7 +143,7 @@ class DeltaGenerator(
     # those, see above.
     def __init__(
         self,
-        container: Optional["BlockPath.ContainerValue"] = BlockPath.MAIN,
+        container: Optional[RootContainer] = RootContainer.MAIN,
```

Comment:
Nit: while we're at it, let's rename the param to root_container too? Better for understanding st.container vs root_container (and also better grepping)

Reply:
It's most confusing in DeltaGenerator since that's also where st.container exists; in other areas like the Cursor constructors I'm more ambivalent (it's more clear, but also more verbose...)

Reply:
OTOH... maybe this is something that type-checking gives to us for mostly free?

Reply:
This is actually something that type checking _won't_ give us if we move to expressing the enum inside a protobuf, because those protobuf enum values are just ints. I haven't made that change yet - I'm about to - but it'll mean that this signature becomes `root_container: Optional[int] = RootContainer.MAIN`

Reply:
(I think this is a reasonable trade-off, and I agree with the parameter name change)